crates/oxc_language_server/src/linter/tester.rs

```diff
@@ -89,11 +92,39 @@ pub struct Tester<'t> {
 
 impl Tester<'_> {
     pub fn new() -> Self {
-        Self { snapshot_suffix: None, server_linter: ServerLinter::new() }
+        Self {
+            snapshot_suffix: None,
+            server_linter: ServerLinter::new(IsolatedLintHandlerOptions {
+                use_cross_module: false,
+                root_path: std::env::current_dir().expect("could not get current dir"),
+            }),
+        }
     }
 
     pub fn new_with_linter(linter: Linter) -> Self {
-        Self { snapshot_suffix: None, server_linter: ServerLinter::new_with_linter(linter) }
+        Self {
+            snapshot_suffix: None,
+            server_linter: ServerLinter::new_with_linter(
+                linter,
+                IsolatedLintHandlerOptions {
+                    use_cross_module: false,
+                    root_path: std::env::current_dir().expect("could not get current dir"),
+                },
+            ),
+        }
+    }
+
+    pub fn new_with_linter_cross_module(linter: Linter) -> Self {
```

Comment:
rather than adding lots of `new_*` functions, should we replace this with a `new_with_server_linter` method, and let the user pass in whatever options and linter they want?

Reply:
To be honest, I do not like the current implementation of the Server Linter and co. The test setup reflects it^^.
Added your change and will look into refactoring.